lib/room.rb

```diff
@@ -1,12 +1,26 @@
-class Rom
+class Room
+attr_reader :category,
+            :length,
+            :width
```

Comment:
you don't need this attr_readers, because we don't need methods for `length` or `width`